src/connectors/hits/__tests__/connectHits-test.js

```diff
@@ -52,7 +55,10 @@ describe('connectHits', () => {
   it('Provides the hits and the whole results', () => {
     const rendering = sinon.stub();
     const makeWidget = connectHits(rendering);
-    const widget = makeWidget();
+    const widget = makeWidget({
+      escapeHits: true,
```

Comment:
What about we use the namespace pattern that we use for other options? This could look like:

```js
const widget = makeWidget({
  escapeHits: true,
});
```

or with the list:

```js
const widget = makeWidget({
  escapeHits: {include: ['title', 'tags']},
});
```